supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

```diff
@@ -737,6 +748,17 @@ IdM uses two types of replication agreements: _domain replication_ agreements, w
 
 *See also*: xref:ostree[OSTree]
 
+[[rex]]
+==== image:images/yes.png[yes] REX (noun, adjective)
+*Description*: In Red{nbsp}Hat Satellite, _REX_ is an acronym for _remote execution_, a feature that allows users to run jobs on hosts from Capsule by using shell scripts or Ansible. For example, a user can enable remote execution (REX) for Capsule and then run a REX job to apply errata to hosts.
```

Comment:
Also, if you are defining the REX acronym in the first sentence, you don't need to redefine it in the second. How about rewriting it like this? 

```suggestion
*Description*: In Red{nbsp}Hat Satellite, _REX_ is an acronym for _remote execution_. You can use REX to run jobs on hosts from Capsule by using shell scripts or Ansible. For example, a user can enable REX for Capsule and then run a REX job to apply errata to hosts.
```